experimental/dds/tree2/src/feature-libraries/chunked-forest/codec/compressedEncode.ts

```diff
@@ -315,8 +361,9 @@ export class NestedArrayShape extends ShapeGeneric<EncodedChunkShape> implements
 		identifiers: DeduplicationTable<string>,
 		shapes: DeduplicationTable<Shape>,
 	): EncodedChunkShape {
+		const shape: EncodedNestedArray = shapes.valueToIndex.get(this.inner.shape) ?? fail("");
```

Comment:
someday we will port assert out of common utils into client, then make the list of APIs run through assert tagging configured per project, then get our fail api to go through tagging so it would get a unique tag for this source location.....